nms/packages/magmalte/app/views/subscriber/SubscriberUtils.js

```diff
@@ -36,6 +36,11 @@ export function convertBitToMbit(val: number) {
   return (val / mBIT).toFixed(2);
 }
 
+export const CoreNetworkTypes = Object.freeze({
+  NT_EPC: 'EPC',
+  NT_5GC: '5GC',
+});
```

Comment:
Unless there's a functional requirement, I would skip the prefix, as it's clear that it's a core network type from context.

```suggestion
export const CoreNetworkTypes = Object.freeze({
  EPC: 'EPC',
  5GC: '5GC',
});
```

Reply:
We need to have a prefix here since for 5GC it can't begin with an integer.